ghost/admin/app/components/custom-theme-settings/boolean.hbs

```diff
@@ -1,18 +1,20 @@
-<div class="gh-stack-item {{if (eq @index 0) "gh-setting-first" "gh-setting"}}">
+<div class="gh-stack-item {{if (eq @index 0) "gh-setting-first" "gh-setting"}} gh-setting-boolean">
     <div class="flex-grow-1">
-        <div class="flex justify-between items-center relative">
-            <span class="gh-setting-title mb0" for={{this.checkboxId}}>
-                {{humanize-setting-key @setting.key}}
-            </span>
-            <div class="for-switch x-small">
+        <div class="flex justify-between items-start relative">
+            <div class="flex flex-column" style="margin-right: 20px;">
```

Comment:
Out of curiosity, do we not have any utility classes for the `margin-right` value that we can use? (i.e `.mr5`)

Reply:
That's a good call!